### DISCUSSION TITLE
Roku Channel Deep linking issue

### DISCUSSION THREAD

**Discussion by johnsonelahi** [rrirower/DP2SceneGraphWizard#30] [General]
<html>
<body>

Channels are required to support roInput events. Please ensure your manifest includes the supports_input_launch=1 entry and your channel properly handles roInput events.
<html>
<body>

Error | DeepLinking | 5.2
-- | -- | --

</body>
</html>
--

<br class="Apple-interchange-newline">
</body>
</html>Error	DeepLinking	[5.2](https://developer.roku.com/docs/developer-program/certification/certification.md#certification-criteria-1)

**Comment by johnsonelahi** [comment]
Please help me to solve the issue.

**Reply by rrirower** [reply]
You have posted in the wrong discussion. The channel feed manager does not convert SDK channels. Please see the DPtoSceneGraph wizard discussions. You'll see that your issue has already been reported. More importantly, the current version of the DPtoSceneGraph wizard software is an Alpha version. This means it should only be used for testing since it is still in development.

**Reply by johnsonelahi** [reply]
Does this mean that the problem will not be solved yet?

**Reply by rrirower** [reply]
Yes. The software is in development. That means there is no timeline for when all problems will be resolved. The goal is to have everything done by the end of the year or sooner.

**Reply by johnsonelahi** [reply]
Can you guide about the issue?

**Reply by rrirower** [reply]
This is not a discussion for this product site. I suggest you go to the DPtoSceneGr aph product page and read the existing discussions.

As I stated, I am in the process of developing the code. Please be patient while I work through the process.

**Comment by rrirower** [comment]
Moved to DPtoSceneGraph project.
